cartesian_enigma/rust/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "cartesian_enigma"
+version = "0.1.0"
+authors = ["Mathieu WILLAY <matt.willay@gmail.com>"]
+edition = "2021"
```

Comment:
Edition 2021 is not release yet, it will be easier to stick to 2018 edition.

If we want to use it anyway, we need to use the `nightly` compiler and add `cargo-features = ["edition2021"]` to the top of the `cargo.toml`

Reply:
Back to 2018 then